### ISSUE TITLE
If we disabled item manually when tooltip is shown, tooltip doesn't hide

### ISSUE THREAD

**Issue by voogryk** [twbs/bootstrap#20150]
Windows 7, Ubuntu 15.10
Chrome 49.0.2623.112+

Expected result:
U hover btn in first 5 sec (before btn became disabled) and we see tooltip. After 5sec btn became disabled and tooltip hide. 

Actual result:
U hover btn in first 5 sec (before btn became disabled) and we see tooltip. After 5sec btn became disabled, but we can see tooltip and if we hover out btn we see tooltip too.

PS when btn became disabled it must be hovered

Example http://jsbin.com/volimenice/1/edit?html,css,js,output

**Comment by twbs-lmvtfy** [comment]
Hi @voogryk!

You appear to have posted a live example (http://jsbin.com/xuhuhitari/1/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 5, column 1: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.2
* line 6, column 1: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.2

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by voogryk** [comment]
http://jsbin.com/hakitecuqe/1/edit?html,css,js,output

**Comment by twbs-lmvtfy** [comment]
Hi @voogryk!

You appear to have posted a live example (http://jsbin.com/hakitecuqe/1/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 5, column 1: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.2
* line 6, column 1: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.2

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by voogryk** [comment]
http://jsbin.com/volimenice/1/edit?html,css,js,output

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
